[docs] Update agent init docs for new interactive flow, skill locations, and dotnet-inspect skill (#827)
microsoft/aspire.dev: src/frontend/src/content/docs/reference/cli/commands/aspire-agent-init.mdx

## Description

The `aspire agent init` command initializes skill files and MCP (Model Context Protocol) server configuration for your development environment. It presents an interactive multi-step flow to configure AI coding agent support:

> 1. **Select skill locations** — choose where skill files are installed (Standard `.agents/skills/`, Claude Code `.claude/skills/`, GitHub Skills `.github/skills/`, OpenCode `.opencode/skill/`). All locations are pre-selected by default.

Reviewer: ```suggestion
1. **Select skill locations** — choose where skill files are installed (Standard `.agents/skills/`, Claude Code `.claude/skills/`, GitHub Skills `.github/skills/`, OpenCode `.opencode/skill/`). The **Standard** location is the only option that defaults as pre-selected.
```
